style/style.css

```diff
@@ -0,0 +1,518 @@
+* {
+  font-family: "Delius Unicase", cursive;
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  background: #ffffff;
+  user-select: none;
+}
+
+.container {
+  max-width: 1170px;
+  margin: 120px auto 120px auto;
+  display: grid;
+  grid-template-columns: 100%;
+  grid-template-rows: fit-content fit-content fit-content;
+  gap: 120px 0px;
+  grid-template-areas:
+    "cities"
+    "nature"
+    "people";
+}
+
+div[class^="image"] {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: cover, cover;
```

Comment:
DRY

Reply:
I couldn't do it...

Reply:
Can't use shorthand with "background-size: cover, cover"

Reply:
shorthand does not work